README.md

```diff
@@ -36,23 +36,31 @@ See [./docs/OffsetHelper.md](./docs/OffsetHelper.md) for detailed documentation.
 
 ## Commands
 
-Use the following commands to compile, test and deploy the contracts:
+Use the following commands to compile and test the Offset Helper:
 
 ```
 yarn compile
 yarn test      # test using a polygon fork
-yarn coverage  # test using a polygon fork with coverage report
-yarn deploy
 ```
 
-Documentation can be auto-generated from the contract's [natspec](https://docs.soliditylang.org/en/latest/natspec-format.html) in [./docs/](./docs/) using
+Deploy the contract locally with:
+
+```
+yarn hardhat --network hardhat deployOffsetHelper --verify false
```

Comment:
Not related to this PR, but I definitely recommend switching this to Hardhat's native approach to deployment.  Tasks were never intended to be used for deployment.  I also recommend using hardhat-deploy to track the artifacts.  This brings a whole bunch of extra benefits.